cmd/performance-profile-creator/cmd/root.go

```diff
@@ -203,17 +210,17 @@ func init() {
 	rootCmd.PersistentFlags().IntVarP(&args.reservedCPUCount, "reserved-cpu-count", "R", 0, "Number of reserved CPUs (required)")
 	rootCmd.MarkPersistentFlagRequired("reserved-cpu-count")
 	rootCmd.PersistentFlags().BoolVarP(&args.splitReservedCPUsAcrossNUMA, "split-reserved-cpus-across-numa", "S", false, "Split the Reserved CPUs across NUMA nodes")
-	rootCmd.PersistentFlags().StringVarP(&args.mcpName, "mcp-name", "T", "worker-cnf", "MCP name corresponding to the target machines (required)")
+	rootCmd.PersistentFlags().StringVarP(&args.mcpName, "mcp-name", "n", "worker-cnf", "MCP name corresponding to the target machines (required)")
```

Comment:
Why not capital letter? it may confuse

Reply:
I made updates based on @fromanirh suggestion [here](https://github.com/openshift-kni/performance-addon-operators/pull/570#discussion_r590338740).

In the previous patch, we used shorthand for topology-manager-policy flag as "Q" (discussion about this [here](https://github.com/openshift-kni/performance-addon-operators/pull/570#discussion_r590322813)) and updated shorthand for mcp-name to "n" (n for name as "N" was already used for policy-name) so I could use "T" for topology-manager-policy.

Please let me know if that is okay? If not please let me know what shorthand letter you would like me to use for topology-manager-policy.

Reply:
I am "lost" a little in letters. I have no special "preference" as long as all are Capital letters (or not)

Reply:
the best option is just to get rid of single-letter options.

Reply:
Sure, can certainly do that if it causes less confusion :)